docs/en/observability/synthetics-intro.asciidoc

```diff
@@ -0,0 +1,70 @@
+[[monitor-uptime-synthetics]]
+= Synthetic monitoring
+
+++++
+<titleabbrev>Synthetics (beta)</titleabbrev>
+++++
+
+beta[]
+
+[NOTE]
+====
+The Synthetics app is for viewing result data from monitors created and managed
+directly in the <<synthetics-get-started-ui,Synthetics UI>> or managed externally
+using <<synthetics-get-started-project,projects>>.
+This can include both lightweight and browser-based monitors, and can include monitors
+running from either Elastic's global managed testing infrastructure or from
+<<synthetics-private-location,Private Locations>>.
+
+To view result data from lightweight monitors running through {heartbeat} and configured with
+a traditional `heartbeat.yml` file, use the <<uptime-intro,{uptime-app}>> instead of the Synthetics app.
+====
```

Comment:
@paulb-elastic @drewpost @florent-leborgne what do you think about this language on the Synthetics intro page to explain the relationship between Synthetics and Uptime?

Reply:
LGTM